lib/arc/actions/validate.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Arc.Actions.Validate do
+
+  defmacro __using__(_) do
+    quote do
+      def valid?(args), do: Arc.Actions.Validate.valid?(__MODULE__, args)
+    end
+  end
+
+  def valid?(definition, {file, scope}) when is_binary(file) or is_map(file) do
+    case Arc.File.new(file) do
```

Comment:
Maybe the answer is to provide a different interface for downloading remote images (as some sort of pre-processing step that the user would do explicitly?)

I'm not really a fan of how it's hidden inside here.  But either way I'd like to think through that use case